dev/tasks/build/builder.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* jshint node: true */
+
+'use strict';
+
+var Builder;
+
+/**
+ * A CKEditor 5 release builder.
+ *
+ * @class Builder
+ */
+module.exports = Builder = function( target ) {
+	/**
+	 * The target directory where to create the build.
+	 *
+	 * **Warning**: if existing, this directory will be deleted before processing.
+	 *
+	 * @type {string}
```

Comment:
We don't use the `@type` tag.

> The use of this tag is deprecated. It's only supported for backwards compatibility with old ext-doc. For new code always use the @property tag instead.

https://github.com/senchalabs/jsduck/wiki/@type